src/main/java/util/JsonHelper.java

```diff
@@ -0,0 +1,89 @@
+package util;
+
+import com.google.gson.*;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.seleniumhq.jetty7.util.ajax.JSON;
+import prefs.Config;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+
+/**
+ * A helper that manages conversion from Config files to JSON Strings, and vice versa.
+ */
+public class JsonHelper {
+    private static final Logger logger = LogManager.getLogger(JsonHelper.class.getName());
+
+    private static final String EMPTY_STRING = "";
+
+    private Gson gson;
+
+    /**
+     * Sets up the JsonHelper by initialising the Gson component
+     */
+    public JsonHelper() {
+        setupGson();
+    }
+
+    /**
+     * Configures the gson object that will serialise and deserialise json data
+     */
+    private void setupGson() {
+        gson = new GsonBuilder()
+                .setPrettyPrinting()
+                .registerTypeAdapter(LocalDateTime.class,
+                        (JsonSerializer<LocalDateTime>) (src, typeOfSrc, context) -> {
+                            Instant instant = src.atZone(ZoneId.systemDefault()).toInstant();
+                            long epochMilli = instant.toEpochMilli();
+                            return new JsonPrimitive(epochMilli);
+                        })
+                .registerTypeAdapter(LocalDateTime.class,
```

Comment:
Never mind, my mistake. It's registering a Serializer and Deserializer with the same method. :unamused: It would be good to factor them out and name them.